clients/src/main/java/org/apache/kafka/clients/admin/ConfigEntry.java

```diff
@@ -174,11 +174,15 @@ public int hashCode() {
         return result;
     }
 
+    /**
+     * Override toString to redact sensitive value.
+     * WARNING, user should be responsible to set correct "senstive" field for each config entry.
```

Comment:
**sensitive** is typo. Also, maybe we can change this sentense: 

user should be responsible to set correct "senstive" field for each config entry.
-> user should be responsible to set **the** correct **"isSensitive"** field for each config entry.

Reply:
Good catch! Fixed as suggested.